src/components/UserSelection.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+
+export default function UserSelection() {
+  const [value, setValue] = React.useState(0);
```

Comment:
`value` is a very generic name

Reply:
Ok got it, changed to `user`, `setUser`

Reply:
I don't see the update

Reply:
This could of been because I did not push, changes should be up now tho you can check.